package.json

```diff
@@ -47,6 +47,7 @@
     "socket.io-client": "^2.4.0",
     "tar-fs": "^1.16.0",
     "tmp-promise": "^1.0.4",
+    "typescript": "^4.2.4",
```

Comment:
Why does node-homey need TS as dependency? Imo it should install TS as dev dependency of the homey app and use that to compile.

Reply:
This is used to invoke tsc

Reply:
Yes but you don't want typescript shipped with node-homey, this would lock the dev on a specific tsc version. Best practice is to install it as dependency in the homey app and run it from there.

Reply:
but tsc is installed in the app we are building, the cli shouldn't need it (except maybe as a fallback but I'm not sure we want/need that)

Reply:
Good point